Adyen.EcommLibrary/Service/Resource/Payment/Authorise.cs

```diff
@@ -11,7 +11,9 @@ public Authorise(AbstractService abstractService)
                     "merchantAccount",
                     "amount.value",
                     "amount.currency",
-                    "reference"
+                    "reference",
+                    "threeDS2RequestData.deviceChannel",

```

Comment:
Those are not required fields. The list contains only required fields

Reply:
Removed on-required fields